src/Http/ResponseEmitter.php

```diff
@@ -162,9 +162,11 @@ protected function emitStatusLine(ResponseInterface $response)
      */
     protected function emitHeaders(ResponseInterface $response)
     {
+        $cookies = $response->cookie();
```

Comment:
Then we should maybe check the response class before calling `cookie()` method to avoid the errors shown by the CI